addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -315,6 +316,80 @@
             </field>
         </record>
 
+        <!-- Inherited form view for mass mailing's form specifically -->
+        <record model="ir.ui.view" id="mailing_mailing_view_full_width_form">
+            <field name="name">mailing.mailing.form_inherit</field>
+            <field name="inherit_id" ref="mass_mailing.view_mail_mass_mailing_form"/>
+            <field name="mode">primary</field>
+            <field name="model">mailing.mailing</field>
+            <field name="arch" type="xml">
+                <xpath expr="//form" position="attributes">
+                    <attribute name="js_class">mass_mailing_full_width_form</attribute>
```

Comment:
Would maybe use something like mailing_mailing_view_form_full_width to standardize naming a bit (and use the same as xml id, so that all naming is coherent and grep is easy)